README.md

```diff
@@ -310,7 +310,7 @@ A maintaner is responsible for merging the PR.
 Say, we have updated dockerfiles/alpine_3.9_2.x and want to check it:
 
 ```sh
-$ IMAGE=tarantool/tarantool TAG=2 OS=alpine DIST=3.9 VER=2.x DOCKERFILE_NAME_SUFFIX=2.x \
+$ IMAGE=tarantool/tarantool TAG=2 OS=alpine DIST=3.9 VER=2.x DOCKERFILE_NAME_SUFFIX= \
```

Comment:
Right, it could be removed, but due to README file gives some usage explanation and not real code, I wanted to save it there as the format of the possible use.

Reply:
Already discussed, just for a note: "This can be saved in the README, but not in this view."